eureka/src/main/java/com/linecorp/armeria/internal/common/eureka/InstanceInfo.java

```diff
@@ -96,6 +98,32 @@ public InstanceInfo(@Nullable @JsonProperty("instanceId") String instanceId,
                         @JsonProperty("dataCenterInfo") DataCenterInfo dataCenterInfo,
                         @JsonProperty("leaseInfo") LeaseInfo leaseInfo,
                         @Nullable @JsonProperty("metadata") Map<String, String> metadata) {
+        this(instanceId, appName, appGroupName, hostName, ipAddr, vipAddress, secureVipAddress, port,
+             securePort, status, homePageUrl, statusPageUrl, null, healthCheckUrl,
+             secureHealthCheckUrl, dataCenterInfo, leaseInfo, metadata);
+    }
+
+    /**
+     * Creates a new instance which may have {@link #healthCheckUrlPath}.
+     */
+    public InstanceInfo(@Nullable String instanceId,
+                        @Nullable String appName,
+                        @Nullable String appGroupName,
+                        @Nullable String hostName,
+                        @Nullable String ipAddr,
+                        @Nullable String vipAddress,
+                        @Nullable String secureVipAddress,
+                        PortWrapper port,
+                        PortWrapper securePort,
+                        InstanceStatus status,
+                        @Nullable String homePageUrl,
+                        @Nullable String statusPageUrl,
+                        @Nullable String healthCheckUrlPath, // Not in JSON
```

Comment:
Global comment. How about just `healthCheckPath`?

Reply:
oh I matched upstream after looking at it. I like healthCheckPath better so fine with changing it back. wdyt?

Reply:
Ah, then let's leave it as is. Thanks!